src/cwe_checker_lib/src/abstract_domain/interval.rs

```diff
@@ -667,7 +667,7 @@ impl RegisterDomain for IntervalDomain {
                 self.clone().sign_extend(width)
             }
             Float2Float | Int2Float | Trunc => IntervalDomain::new_top(width),
-            PopCount => {
+            PopCount | LzCount => {
```

Comment:
Well, it is not fully monotonically decreasing because the interval can contain negative numbers. But you are still right for positive numbers and I improved the computation accordingly.